packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1711,6 +1719,42 @@ QUnit.module('options changed', moduleSetup, () => {
     });
 });
 
+QUnit.module('focus on deleting', () => {
+    const items = ['item 1', 'item 2', 'item 3'];
+    QUnit.test('first item should not be focused after deleting when focusStateEnabled = false (T1226030)', function(assert) {
+        const $element = $('#list').dxList({
+            items,
+            focusStateEnabled: false,
+            allowItemDeleting: true,
+            itemDeleteMode: 'static',
+        });
+        let $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+        const firstItemIsFocused = $listItems.eq(0).hasClass(`${LIST_FOCUS_STATE_CLASS}`);
+
+        $listItems.eq(0).find(`.${STATIC_DELETE_BUTTON_CLASS}`).trigger('dxclick');
+        $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
```

Comment:
excess

Reply:
This is necessary to be duplicated since the first declaration of listItems is a listItems group that is not updated yet i declared it again to get the new updated listItems after button click...

Reply:
This is necessary to be duplicated since the first declaration of listItems is a listItems group that is not updated yet i declared it again to get the new updated listItems after button click...